### ISSUE TITLE
mod downloading is borked

### ISSUE THREAD

**Issue by Caliel666** [Brod8362/d4m#24]
it was working fine before, nothing changed in d4m, happens on both sites
```shell
Failed to install Egg's Songpack(ModManager.install_mod() missing 1 required positional argument: 'category')
Traceback (most recent call last):
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/tui.py", line 72, in menu_install
    mod_manager.install_mod(mod['id'], origin=mod['origin'])
TypeError: ModManager.install_mod() missing 1 required positional argument: 'category'

**Comment by Brod8362** [comment]
Thanks. Fixed in unstable.

**Comment by Brod8362** [comment]
Fixed in v0.3.3.
